pipeline/terraform/main.tf

```diff
@@ -35,3 +35,43 @@ module "catalogue_pipeline_20190426" {
   vhs_sierra_read_policy = "${local.vhs_sierra_read_policy}"
   vhs_miro_read_policy   = "${local.vhs_miro_read_policy}"
 }
+
+module "catalogue_pipeline_20190723" {
+  source = "stack"
+
+  namespace = "catalogue-20190723"
+
+  release_label = "prod_es7"
+
+  account_id = "${data.aws_caller_identity.current.account_id}"
+  aws_region = "${local.aws_region}"
+  vpc_id     = "${local.vpc_id}"
+  subnets    = ["${local.private_subnets}"]
+
+  dlq_alarm_arn = "${local.dlq_alarm_arn}"
+
+  # Transformer config
+  #
+  # If this pipeline is meant to be reindexed, remember to uncomment the
+  # reindexer topic names.
+
+  sierra_adapter_topic_names = [
+    "${local.sierra_reindexer_topic_name}",
+    "${local.sierra_merged_bibs_topic_name}",
+
+    "${local.sierra_merged_items_topic_name}",
+  ]
+  miro_adapter_topic_names = [
+    "${local.miro_reindexer_topic_name}",
+    "${local.miro_updates_topic_name}",
+  ]
+  # Elasticsearch
+  es_works_index = "v2-2019-04-26-contributors-label-from-multiple-subfields"
```

Comment:
nit: is this index name correct? Doesn't match the date.

Reply:
When we created the new ES deployment - it copies everything over, so it's not really a new index, it's a new ES deployment with V7 of the index.